charts/catalog/values.yaml

```diff
@@ -54,3 +54,5 @@ controllerManager:
   verbosity: 10
   # Resync interval; format is a duration (`20m`, `1h`, etc)
   resyncInterval: 5m
+  # Broker relist interval; format is a duration (`20m`, `1h`, etc)
+  brokerRelistInterval: 30m
```

Comment:
24 hours is a better default for production, 30 minutes is better for dev.  It seems to be throwing people, so I'll just make them the same.